lib/ProgressBar/ProgressBar.less

```diff
@@ -26,6 +26,7 @@
 /* Popup */
 .moon-progress-bar-popup {
 	white-space: nowrap;
+	border-radius: 72px;
```

Comment:
Do you know how you came up with this specific number, vs. using a value similar to what we're using for Button?